### PR TITLE
Add a demo and update docs

### LINKED ISSUE #70: Update demo instructions
## Overview

The "Try It" section of the README as well as the Quick Start guide in the docs need to be updated.  There no longer is a doc or doc/demo-checks.  We should add another demo, kick-the-tires set of f/c's and update the documentation accordingly or remove the "Try It" section from the README and any refs to the demo in the quick start docs.

## Requirements

See overview

## Approach

TBD

## Security and Privacy

N/A

## Test Plan

- If implementing a demo set of f/c's, instructions should work

### PR DESCRIPTION
**Pull request by alfinkel**
- [x] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](../blob/master/DCO1.1.txt)

## What

Add a demo and update docs

## Why

To make new users comfortable with a demo implementation and quick start guide

## How

Add a demo implementation and update README and Quick Start guide

## Test

Demo runs

## Context

Closes #70

### CODE REVIEW

File: doc-source/quick-start.rst [hunk 1]
```diff
@@ -5,124 +5,160 @@
...
+~~~~~~~~~~~~~~~~~~~~~~
 
-You might want to push the evidences generated by ``--fetch`` into a
-remote git repository. For that, you can use ``--evidence`` and ``-C
-setup.json``. This ``setup.json`` contains a fake URL of the repo, you
-can edit it to match the URL you want. However, you have to provide
-GitHub credentials. By default, you should use
-``~/.credentials`` (or use ``--creds-path`` to point somewhere else)::
+Storing and using evidence in a Git remote hosting service repository is achievable
+by using the ``--evidence`` option with settings of either ``no-push`` or
```
Comment (drsm79) [inline doc-source/quick-start.rst:71]: Make it clear that no push only retrieves

Comment (alfinkel) [reply doc-source/quick-start.rst:71]: done in https://github.com/ComplianceAsCode/auditree-framework/pull/77/commits/cb7696375a7fdbe40a96a450056725ed0dc40852

File: doc-source/quick-start.rst [hunk 2]
```diff
@@ -5,124 +5,162 @@
...
-The Auditree framework requires, at least, one directory with the
-fetchers and checks.
+The Auditree framework requires at least one directory containing fetchers and
+checks.  Let's use the fetchers and checks packaged up in the ``demo`` folder
+of the ``auditree-framework`` project as an example.  First things first, we need
+to ensure that we have a proper Python virtual environment configured and running::
 
-A normal execution is broken down in 2 steps:
+  $ cd demo
+  $ virtualenv venv
```
Comment (drsm79) [inline doc-source/quick-start.rst:17]: Use python -m venv venv here

File: doc-source/quick-start.rst [hunk 3]
```diff
@@ -5,124 +5,162 @@
...
+mode which will not push the evidence locker to the remote evidence locker::
 
-If you really want to push the evidences generated::
+  $ compliance --check demo.arboretum.accred,demo.custom.accred --evidence no-push -C auditree_demo.json
 
-  $ compliance --fetch --evidence full-remote -C setup.json .
 
-Checks
-~~~~~~
+To execute checks in ``full-remot`` mode which will push the evidence locker to
```
Comment (drsm79) [inline doc-source/quick-start.rst:106]: full-remote (missing e)

**Comment by drsm79** [comment]
This looks great. I wonder if we should rename https://github.com/auditree/auditree-tekton-example and have it hold a full configuration or the demo stuff. So at the end of the prose we can say “here’s one we made earlier”.

File: doc-source/quick-start.rst (hunk 2 above)
Comment (alfinkel) [reply doc-source/quick-start.rst]: done in 61ea3bc

File: doc-source/quick-start.rst (hunk 3 above)
Comment (alfinkel) [reply doc-source/quick-start.rst]: fixed in 6d0bc94

**Comment by drsm79** [comment]
Have moved the repo to https://github.com/auditree/auditree-demo and set it as a template repo, which means others can use it as a base. I’ll put the example config in there later today.

**Review by drsm79 — DISMISSED** [review]

**Review by drsm79 — APPROVED** [review]

**Pull request merged**